pkg/client/scanner/scanner.go

```diff
@@ -31,44 +35,52 @@ type Config struct {
 }
 
 // Available list all available scanner tools that can be used.
-var Available = [...]Tool{
-	Licensee,
-	Scancode,
-	Composer,
-	FileHasher,
-}
+var Available []Tool
 
 // Default is the default scanner tools that shall be used if no particular tool is selected.
-var Default Tool = Licensee
+var Default Tool
+
+// init loads available scanner tools from config file. It also assigns default tool
+func init() {
+
+	jsonFile, err := os.Open("pkg/client/scanner/scanner_tools.json")
+	if err != nil {
+		fmt.Println(err)
+		jsonFile, err = os.Open("./scanner_tools.json")
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(0)
+		}
+	}
+	byteValue, _ := ioutil.ReadAll(jsonFile)
+	defer jsonFile.Close()
+
+	err = json.Unmarshal(byteValue, &Available)
+	if err != nil {
+		log.Println(err)
+	}
 
-// Licensee represents the latest usable Licensee version
-var Licensee = Tool{
-	Name:      "Licensee",
-	Version:   "9.13.0",
-	DockerImg: "docker.pkg.github.com/osrgroup/product-model-toolkit/scanner-licensee:9.13.0",
-	Cmd:       `/bin/bash -c "licensee detect /input/ --json > /result/result.json"`,
-	Results:   []string{"result.json"},
+	Default = Available[0]
 }
 
-// Scancode represents the latest usable Scancode version
-var Scancode = Tool{
-	Name:      "Scancode",
-	Version:   "3.1.1",
-	DockerImg: "docker.pkg.github.com/osrgroup/product-model-toolkit/scanner-scancode:3.1.1",
-	Cmd:       `/bin/bash -c "./scancode --spdx-tv /result/result.spdx /input"`,
-	Results:   []string{"result.spdx"},
+// NoTools returns true if no scanner tools available
+func NoTools() bool {
+	if len(Available) != 0 {
+		return false
+	}
+	return true
 }
 
-// FromStr returns a tool with the given name. If unable to find a tool with the given name it returns the default tool.
-func FromStr(name string) Tool {
+// FromStr returns a tool with the given name. If no tool found, bool return value is set to false
```

Comment:
Change comment to
`FromStr returns a tool with the given name and indicates with a bool if the tool could be found.`